clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1116,8 +1119,8 @@ def create_adni_scans_files(conversion_path, bids_subjs_paths):
                     file_name = os.path.basename(file)
                     scans_df["filename"] = path.join(mod_name, file_name)
                     converted_mod = find_conversion_mod(file_name)
-                    conversion_df = converted_dict[converted_mod]
                     try:
+                        conversion_df = converted_dict[converted_mod]
```

Comment:
Could you move that line again ? (1123 to 1119)

Reply:
Yes I will move conversion_df to above the try statement